### DISCUSSION TITLE
TTS my own voice (help out a noob :// )

### DISCUSSION THREAD

**Discussion by IbrahimOzmen** [coqui-ai/TTS#4077] [General Q&A]
I need to tts my own voice with a text input. i have tried it on the xtts space but it did not meet my expactations. i need better voice quality. what i am wondering is can i fine tune this model with my own voice data. if i can how many hours of voice data do i need? I hope i have explained myself well. Can you leave me a fine tune documentation or link suitable for this special request?

**Comment by eginhard** [comment]
https://coqui-tts.readthedocs.io/en/latest/models/xtts.html#training

**Comment by IbrahimOzmen** [comment]
i tried to run the google colab notebook but in the first code block it gave lots of errors. and in the second block gradio gui link didnt appear. do u know what i should do or can you send me a working link? thanks so much

**Reply by eginhard** [reply]
Right, if you want to use collab/gradio, best to use this one, the other one is not updated anymore: https://github.com/daswer123/xtts-finetune-webui

**Reply by IbrahimOzmen** [reply]
in this colab notebook, when i try to create dataset in the first page of gradio, it "gives connection errored out." and on the interface code block output is like this:

model.bin: 91% 2.82G/3.09G [01:06<00:06, 41.7MB/s]
model.bin: 92% 2.83G/3.09G [01:06<00:06, 41.9MB/s]
model.bin: 92% 2.84G/3.09G [01:06<00:05, 42.9MB/s]
model.bin: 92% 2.85G/3.09G [01:07<00:05, 41.6MB/s]
model.bin: 93% 2.86G/3.09G [01:07<00:05, 41.9MB/s]
model.bin: 93% 2.87G/3.09G [01:07<00:05, 42.1MB/s]
model.bin: 93% 2.88G/3.09G [01:07<00:04, 42.1MB/s]
model.bin: 94% 2.89G/3.09G [01:08<00:04, 41.6MB/s]
model.bin: 94% 2.90G/3.09G [01:08<00:04, 42.0MB/s]
model.bin: 94% 2.92G/3.09G [01:08<00:04, 42.4MB/s]
model.bin: 95% 2.93G/3.09G [01:08<00:03, 42.5MB/s]
model.bin: 95% 2.94G/3.09G [01:09<00:03, 41.3MB/s]
model.bin: 95% 2.95G/3.09G [01:09<00:03, 41.8MB/s]
model.bin: 96% 2.96G/3.09G [01:09<00:03, 35.1MB/s]
model.bin: 96% 2.98G/3.09G [01:10<00:02, 43.6MB/s]
model.bin: 97% 2.99G/3.09G [01:10<00:02, 43.2MB/s]
model.bin: 97% 3.00G/3.09G [01:10<00:02, 43.1MB/s]
model.bin: 97% 3.01G/3.09G [01:10<00:01, 42.8MB/s]
model.bin: 98% 3.02G/3.09G [01:11<00:01, 42.2MB/s]
model.bin: 98% 3.03G/3.09G [01:11<00:01, 42.3MB/s]
model.bin: 99% 3.04G/3.09G [01:11<00:01, 42.5MB/s]
model.bin: 99% 3.05G/3.09G [01:11<00:00, 42.4MB/s]
model.bin: 99% 3.06G/3.09G [01:12<00:00, 41.8MB/s]
model.bin: 100% 3.07G/3.09G [01:12<00:00, 41.9MB/s]
model.bin: 100% 3.09G/3.09G [01:12<00:00, 42.5MB/s]
Existing language matches target language
Unable to load any of {libcudnn_ops.so.9.1.0, libcudnn_ops.so.9.1, libcudnn_ops.so.9, libcudnn_ops.so}
Invalid handle. Cannot load symbol cudnnCreateTensorDescriptor

i asked gpt about this and it said it must be something about cuda and cuddn version. cuda v12.6 is installed on my pc right now but dunno which version is compatable with this colab notebook. please any help will be appreciated
